### PR TITLE
[14.0][IMP] storage_image_product: add missing index

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

File: storage_image_product/models/product_image_relation.py [hunk 1]
```diff
@@ -11,7 +11,9 @@ class ProductImageRelation(models.Model):
     _description = "Product Image Relation"
 
     attribute_value_ids = fields.Many2many(
-        "product.attribute.value", string="Attributes"
+        "product.attribute.value",
+        string="Attributes",
+        index=True,
```
Comment (simahawk) [inline storage_image_product/models/product_image_relation.py]: does this have any effect on m2m?

Comment (sebastienbeau) [reply storage_image_product/models/product_image_relation.py]: you are right this one was useless. Thx

**Review by simahawk — APPROVED** [review]

**Review by hparfr — APPROVED** [review]

**Comment by simahawk** [comment]
for the commit msg same remark as on https://github.com/OCA/storage/pull/173 :stuck_out_tongue_winking_eye:

**Pull request merged**